RadarSDK/RadarAddress+Internal.h

```diff
@@ -30,10 +30,13 @@
                                       county:(NSString *_Nullable)county
                                 neighborhood:(NSString *_Nullable)neighborhood
                                       number:(NSString *_Nullable)number
+                                      street:(NSString *_Nullable)street
```

Comment:
Very surprised we weren't storing this previously. Makes me wonder if I'm missing something...